app/assets/stylesheets/layout_components/credit-card.scss

```diff
@@ -0,0 +1,34 @@
+.credit-card{
+  background-color: $cerulean;
+  border-radius: 3px;
+  padding: $base-padding;
+  max-width: 400px;
+  min-height: 216px;
+  position: relative;
+
+  &__logo-visa{
+    background: url(image-path("cc-type-visa-white.png")) no-repeat;
+    background-size: contain;
+    height: 50px;
+    width: 100px;
+    float:right;
+  }
+
+  &__logo-mc{
```

Comment:
no, a mixin isn't the best way BUT there is a better way to do this, i'll make the shared styles be for the element logo, and then i'll create modifiers containing the logos for each one!